### PR TITLE
Iss hipm 910 As a user, I want to use the application in English

### PR DESCRIPTION
**Pull request by Senn3**
I have created a new resource file for the english language and added some scripts, which set the used app language to the smartphone language. This is done in the same way as the xamarin sample code for localization (https://github.com/xamarin/xamarin-forms-samples/tree/master/UsingResxLocalization).
This can be tested by switching the smartphone language from english to german or the other way around (on Android: Settings -> Languages & input -> Languages)

### CODE REVIEW

**Comment by sayantik** [comment]
I'm not sure of the likelihood of the scenario but my default language was Spanish when I installed the app, then should the app content be in English or German? Currently it shows in English. @Resudo what do you think?

**Comment by Resudo** [comment]
@sayantik The default language for all system languages but German should be English. A person using a system language different from German and English is in my opinion more likely to understand English than German.

**Comment by Resudo** [comment]
@Senn3 @sayantik Before this pull request gets merged, a person experienced in both English and German should check if all the translations are appropriate for our application. If you don't want to bother anyone else with this task, I can do it myself over the weekend. Just request a PR in that case.

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 1]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="RouteDetailsPage_Tags" xml:space="preserve">
-    <value>Auf der Route</value>
+  <data name="RouteDetailsPage_Tags" xml:space="preserve">
+    <value>On the route</value>
   </data>
-	<data name="RoutesOverviewListItemViewModel_Distance" xml:space="preserve">
-    <value>{0} Kilometer</value>
+  <data name="RoutesOverviewListItemViewModel_Distance" xml:space="preserve">
+    <value>{0} kilometre</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: We are talking about the plural of _kilometre_ which imo should be _kilometres_.

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 2]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="SettingsScreenView_AutoStartAudio_Text" xml:space="preserve">
-    <value>Wenn diese Einstellung aktiviert ist, wird Audio automatisch für die aktuelle Seite gestartet.</value>
+  <data name="SettingsScreenView_AutoStartAudio_Text" xml:space="preserve">
+    <value>When this setting is enabled, audio will automatically start for the current page.</value>
   </data>
-	<data name="SettingsScreenView_AutoStartAudio_Title" xml:space="preserve">
-    <value>Automatisch Audio beginnen</value>
+  <data name="SettingsScreenView_AutoStartAudio_Title" xml:space="preserve">
+    <value>Automatically Start Audio</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Automatically start audio_ - no caps

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 3]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="SettingsScreenView_AutoSwitchPage_Text" xml:space="preserve">
-    <value>Wenn diese Einstellung aktiviert ist, wird nach Ende der Audiowiedergabe automatisch zur nächsten Seite gewechselt.</value>
+  <data name="SettingsScreenView_AutoSwitchPage_Text" xml:space="preserve">
+    <value>When this setting is enabled, the next page will be automatically shown after the end of an audio file.</value>
   </data>
-	<data name="SettingsScreenView_AutoSwitchPage_Title" xml:space="preserve">
-    <value>Automatisch Seiten wechseln</value>
+  <data name="SettingsScreenView_AutoSwitchPage_Title" xml:space="preserve">
+    <value>Automatically Change Pages</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Automatically change pages_ - no caps

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 4]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="AudioToolbarView_Progress" xml:space="preserve">
-    <value>Fortschritt</value>
+  <data name="AudioToolbarView_Progress" xml:space="preserve">
+    <value>Progress</value>
   </data>
-	<data name="SettingsScreenView_AppIntro_Title" xml:space="preserve">
-    <value>App Einführung</value>
+  <data name="SettingsScreenView_AppIntro_Title" xml:space="preserve">
+    <value>App Introduction</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _App introduction_ - no caps

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 5]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="UserOnboarding_Students_Title" xml:space="preserve">
-    <value>Von Studierenden</value>
+  <data name="UserOnboarding_Students_Text" xml:space="preserve">
+    <value>An evolving app created by students of the University of Paderborn</value>
   </data>
-	<data name="LoadingPage_Subtext" xml:space="preserve">
-    <value>Einen Moment bitte...</value>
+  <data name="UserOnboarding_Students_Title" xml:space="preserve">
+    <value>Made By Students</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Made by students_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 6]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="LoadingPage_Text" xml:space="preserve">
-    <value>Historisches Paderborn wird geladen</value>
+  <data name="LoadingPage_Subtext" xml:space="preserve">
+    <value>One moment please...</value>
   </data>
-	<data name="MapRenderer_NoLocation_Text" xml:space="preserve">
-    <value>Sie sind nicht in Paderborn oder es ist momentan kein GPS Signal vorhanden</value>
+  <data name="LoadingPage_Text" xml:space="preserve">
+    <value>Historical Paderborn is going to be loaded</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: I fell like _Historical Paderborn is being loaded_ would be a better fit here.

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 7]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="ExhibitDetailsPage_AdditionalInformation" xml:space="preserve">
-    <value>Zusätzliche Informationen</value>
+  <data name="RouteDetailspageViewModel_RouteStarted" xml:space="preserve">
+    <value>The route has already started:</value>
   </data>
-	<data name="ExhibitNearby_ExhibitNearby" xml:space="preserve">
-    <value>Sehenwürdigkeit in der Nähe</value>
+  <data name="ExhibitDetailsPage_AdditionalInformation" xml:space="preserve">
+    <value>Additional Informations</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Additional information_ - no caps and information is already the plural

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 8]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="UserOnboarding_Route_Title" xml:space="preserve">
-    <value>Themenspezifische Routen</value>
+  <data name="UserOnboarding_Route_Text" xml:space="preserve">
+    <value>Explore Paderborn on your own or with topic-specific routes</value>
   </data>
-	<data name="UserOnboarding_Skip" xml:space="preserve">
-    <value>ÜBERSPRINGEN</value>
+  <data name="UserOnboarding_Route_Title" xml:space="preserve">
+    <value>Theme-specific Routes</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Theme-specific routes_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 9]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="SkipExhibit_Title" xml:space="preserve">
-    <value>Sehenswürdigkeit überspringen</value>
+  <data name="SkipExhibit_Question_Part2" xml:space="preserve">
+    <value>?</value>
   </data>
-	<data name="SettingsScreenView_FurtherSettings_Title" xml:space="preserve">
-    <value>Weitere Einstellungen</value>
+  <data name="SkipExhibit_Title" xml:space="preserve">
+    <value>Skip Sight</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Skip sight_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 10]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="SettingsScreenView_RemoveAllDownloadsButton_Text" xml:space="preserve">
-    <value>Entfernen</value>
+  <data name="SettingsScreenView_FurtherSettings_Title" xml:space="preserve">
+    <value>More Settings</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _More settings_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 11]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="SettingsScreenView_RemoveAllDownloadsPrompt_Question" xml:space="preserve">
-    <value>Möchten Sie wirklich alle Downloads entfernen? Die App muss dafür neu gestartet werden.</value>
+  <data name="SettingsScreenView_RemoveAllDownloadsPrompt_Confirm" xml:space="preserve">
+    <value>Remove</value>
   </data>
-	<data name="SettingsScreenView_RemoveAllDownloadsPrompt_Title" xml:space="preserve">
-    <value>Alle Downloads entfernen</value>
+  <data name="SettingsScreenView_RemoveAllDownloadsPrompt_Question" xml:space="preserve">
+    <value>Do you really want to remove  all downloads? The app has to be restarted for this.</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: There appear to be 2 blank spaces between _remove_ and _all_.

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 12]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="SettingsScreenView_RemoveAllDownloads_Text" xml:space="preserve">
-    <value>Alle Downloads entfernen (Nutzen Sie diese Option, um Speicherplatz freizugeben)</value>
+  <data name="SettingsScreenView_RemoveAllDownloadsPrompt_Title" xml:space="preserve">
+    <value>Remove All Downloads</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Remove all downloads_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 13]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="DownloadData_Title" xml:space="preserve">
-    <value>Neue Daten herunterladen?</value>
+  <data name="RoutesOverviewView_EmptyList" xml:space="preserve">
+    <value>No routes available. Download routes to get started!</value>
   </data>
-	<data name="DownloadData_Always" xml:space="preserve">
-    <value>Immer</value>
+  <data name="DownloadData_Title" xml:space="preserve">
+    <value>Donwload new data?</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: Typo already fixed?

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 14]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="StorageTitle" xml:space="preserve">
-    <value>Belegter Speicherplatz für Daten</value>
+  <data name="LoadingPageViewModel_BaseData_DownloadViaMobile" xml:space="preserve">
+    <value>It is recommended to keep the base data up to date. The settings define that data should not be downloaded via mobile data connections. Do you still want to download this base data anyway?</value>
   </data>
-	<data name="DownloadDetails_Text_Part1" xml:space="preserve">
-    <value>Lade Daten zu </value>
+  <data name="StorageTitle" xml:space="preserve">
+    <value>Used Disk Space for Data</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Used disc space for data_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 15]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="LoginScreenView_Email" xml:space="preserve">
-    <value>E-Mail</value>
+  <data name="LoginScreenView_Email" xml:space="preserve">
+    <value>E-mail</value>
   </data>
-	<data name="LoginScreenView_Forgot_Password" xml:space="preserve">
-    <value>Passwort vergessen?</value>
+  <data name="LoginScreenView_Forgot_Password" xml:space="preserve">
+    <value>Forgot Password?</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Forgot password_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 16]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="ForgotPasswordScreenView_Email" xml:space="preserve">
-    <value>E-Mail</value>
+  <data name="ForgotPasswordScreenView_Email" xml:space="preserve">
+    <value>E-mail</value>
   </data>
-	<data name="ForgotPasswordScreenView_Forgot_Password" xml:space="preserve">
-    <value>Passwort vergessen?</value>
+  <data name="ForgotPasswordScreenView_Forgot_Password" xml:space="preserve">
+    <value>Forgot Password?</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Forgot password_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 17]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="RegisterScreenView_Alert_Description" xml:space="preserve">
-    <value>Eine E-Mail wurde Ihnen gesendet. Bitte bestätigen Sie diese E-Mail, um sich einloggen zu können.</value>
+  <data name="RegisterScreenView_Alert_Description" xml:space="preserve">
+    <value>An e-mail has been sent to you. Please confirm this e-mail to be able to log in.</value>
   </data>
-	<data name="RegisterScreenView_Alert_Registered" xml:space="preserve">
-    <value>Benutzer Registrierung</value>
+  <data name="RegisterScreenView_Alert_Registered" xml:space="preserve">
+    <value>User Registration</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _User registration_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 18]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="Alert_Confirm" xml:space="preserve">
-    <value>Bestätigen</value>
+  <data name="Alert_Confirm" xml:space="preserve">
+    <value>Confirm</value>
   </data>
-	<data name="Alert_No_Internet_Title" xml:space="preserve">
-    <value>Keine Netzwerkverbindung</value>
+  <data name="Alert_No_Internet_Title" xml:space="preserve">
+    <value>No Network Connection</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _No network connection_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 19]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="Alert_Unknown_Error_Description" xml:space="preserve">
-    <value>Es ist ein unbekannter Fehler aufgetreten.</value>
+  <data name="Alert_Unknown_Error_Description" xml:space="preserve">
+    <value>An unknown error occurred.</value>
   </data>
-	<data name="Alert_Unknown_Error_Title" xml:space="preserve">
-    <value>Unbekannter Fehler</value>
+  <data name="Alert_Unknown_Error_Title" xml:space="preserve">
+    <value>Unknown Error</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Unknown error_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 20]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="RegisterScreenView_Error_Mismatch_Password" xml:space="preserve">
-    <value>Die Passwörter stimmen nicht überein</value>
+  <data name="RegisterScreenView_Error_Mismatch_Password" xml:space="preserve">
+    <value>The passwords are not matching</value>
   </data>
-	<data name="RegisterScreenView_Error_Invalid_Email" xml:space="preserve">
-    <value>Ungültige E-Mail</value>
+  <data name="RegisterScreenView_Error_Invalid_Email" xml:space="preserve">
+    <value>Invalid E-Mail</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Invalid e-mail_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 21]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="ExhibitDetailsPage_Distance_Text" xml:space="preserve">
-    <value>Sie müssen die Sehenswürdigkeit zuerst besucht haben, um die Details zu sehen!</value>
+  <data name="ExhibitDetailsPage_Distance_Text" xml:space="preserve">
+    <value>You have to visit the sight first before the details are available!</value>
   </data>
-	<data name="ExhibitDetailsPage_Distance_Title" xml:space="preserve">
-    <value>Nicht freigeschaltet</value>
+  <data name="ExhibitDetailsPage_Distance_Title" xml:space="preserve">
+    <value>Not Unlocked</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Not unlocked_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 22]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
 	<data name="ProfileView_Professor" xml:space="preserve">
     <value>Professor</value>
   </data>
-	<data name="ExhibitRouteDownloadPageViewModel_Wifi_Only_Message" xml:space="preserve">
-    <value>In den Einstellungen ist festgelegt, dass Daten nur über Wlan heruntergeladen werden sollen. Sollen die Daten dieses Mal trotzdem über die mobile Datenverbindung heruntergeladen werden?</value>
+  <data name="ExhibitRouteDownloadPageViewModel_Wifi_Only_Message" xml:space="preserve">
+    <value>The settings specify that data should only be downloaded via WiFi. Should the data still be downloaded via the mobile data connection this time?</value>
+  </data>
+  <data name="ExhibitRouteDownloadPageViewModel_Wifi_Only_Title" xml:space="preserve">
+    <value>WiFi Disabled</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _WiFi disabled_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 23]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="ExhibitRouteDownloadPageViewModel_Wifi_Only_Title" xml:space="preserve">
-    <value>Wlan deaktiviert</value>
+  <data name="ForgotPasswordScreenView_Alert_No_Connection_Description" xml:space="preserve">
+    <value>Please check your network connection</value>
   </data>
-	<data name="ForgotPasswordScreenView_Alert_No_Connection_Description" xml:space="preserve">
-    <value>Bitte überprüfen Sie ihre Netzwerkverbindung</value>
+  <data name="ForgotPasswordScreenView_Alert_No_Connection_Title" xml:space="preserve">
+    <value>No Network Connection</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _No network connection_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 24]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="ForgotPasswordScreenView_Alert_No_Connection_Title" xml:space="preserve">
-    <value>Keine Netzwerkverbindung</value>
+  <data name="ForgotPasswordScreenView_Alert_Unknown_Error_Description" xml:space="preserve">
+    <value>An unknown error occurred. Please try again later.</value>
   </data>
-	<data name="ForgotPasswordScreenView_Alert_Unknown_Error_Description" xml:space="preserve">
-    <value>Es ist ein unbekannter Fehler aufgetreten. Bitte versuchen Sie es später noch einmal.</value>
+  <data name="ForgotPasswordScreenView_Alert_Unknown_Error_Title" xml:space="preserve">
+    <value>Unknown Error</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Unknown error_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 25]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="ExhibitNearby_VisitRequest" xml:space="preserve">
-    <value>Besuche eine nahe Sehenswürdigkeit!</value>
+  <data name="UserRating_Rate_Count" xml:space="preserve">
+    <value>Ratings</value>
   </data>
-	<data name="UserRating_Rate_Count" xml:space="preserve">
-    <value>Bewertungen</value>
+  <data name="UserRating_Average_Rating" xml:space="preserve">
+    <value>Average Ratings</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Average ratings_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 26]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="UserRating_Dialog_Message_No_Internet" xml:space="preserve">
-    <value>Es kann keine Verbindung zum Internet aufgebaut werden.</value>
+  <data name="UserRating_Dialog_Message_No_Rating" xml:space="preserve">
+    <value>None of the stars were marked for a rating.</value>
   </data>
-	<data name="UserRating_Dialog_Message_No_Rating" xml:space="preserve">
-    <value>Es wurde keiner der Sterne zum Bewerten markiert.</value>
+  <data name="UserRating_Dialog_Title_Not_Logged_In" xml:space="preserve">
+    <value>Not Logged In</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Not logged in_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 27]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="UserRating_Dialog_Title_Not_Logged_In" xml:space="preserve">
-    <value>Nicht angemeldet</value>
+  <data name="UserRating_Dialog_Title_No_Internet" xml:space="preserve">
+    <value>No Internet Connection</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _No internet connection_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 28]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="UserRating_Dialog_Title_No_Internet" xml:space="preserve">
-    <value>Keine Internetverbindung</value>
+  <data name="UserRating_Dialog_Title_No_Rating" xml:space="preserve">
+    <value>No Rating</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _No rating_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 29]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="UserRating_Dialog_Message_Thx" xml:space="preserve">
-    <value>Vielen Dank für Ihre Bewertung.</value>
+  <data name="UserRating_Dialog_Message_Unkown_Error" xml:space="preserve">
+    <value>An unknown error has occurred. Please try again later.</value>
   </data>
-	<data name="UserRating_Dialog_Message_Unkown_Error" xml:space="preserve">
-    <value>Es ist ein unbekannter Fehler aufgetreten. Bitte versuchen Sie es zu einem späteren Zeitpunkt erneut.</value>
+  <data name="UserRating_Dialog_Title_Thx" xml:space="preserve">
+    <value>Thank You</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Thank you_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 30]
```diff
@@ -193,668 +193,669 @@ Software Innovation Campus Paderborn
...
   </data>
-	<data name="UserRating_Dialog_Title_Thx" xml:space="preserve">
-    <value>Dankeschön</value>
+  <data name="UserRating_Dialog_Title_Unkown_Error" xml:space="preserve">
+    <value>Unknown Error</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Unknown error_

File: Sources/HipMobileUI/Resources/Strings.resx [hunk 31]
```diff
@@ -863,12 +864,12 @@ Verantwortlich für den Betrieb: Dr. Simon Oberthür</value>
     <value>Check other mode</value>
   </data>
 	<data name="CharacterSelectionPage_SelectCharacter" xml:space="preserve">
-    <value>Wähle einen Charakter</value>
+    <value>Choose a Character</value>
```
Comment (Resudo) [inline Sources/HipMobileUI/Resources/Strings.resx]: _Choose a character_

**Pull request merged**